plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -200,7 +220,7 @@ private class DokkaDescriptorVisitor( // TODO: close this class and make it priv
             companion = descriptor.companion(driWithPlatform),
             supertypes = PlatformDependent.from(platformData, info.supertypes),
             platformData = listOf(platformData),
-            extra = descriptor.additionalExtras()
+            extra = toContainer(descriptor.additionalExtras(), descriptor.getAnnotations())
```

Comment:
You can write it as `PropertyContainer.empty() + descriptor.additionalExtras() + descriptor.getAnnotations()` or create a `addAll()` method

Reply:
Added.